credit-card-recommendation/notebook/primehub_sdk_credit_card_recommendation.ipynb

```diff
@@ -0,0 +1,226 @@
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "fdd63072",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# Please filled in the PrimeHub cluster.\n",
+    "PRIMEHUB_CLUSTER = <primehub-cluster-url>"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "77a1fa47",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# Please filled in the kaggle config.\n",
+    "KAGGLE_CONFIG = <kaggle-config-url>"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "4940ea7d",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# Setting the primehub sdk config.\n",
+    "from primehub import PrimeHub, PrimeHubConfig\n",
+    "ph = PrimeHub(PrimeHubConfig())"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "b4361871",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# Login config setting\n",
+    "import os\n",
+    "if not os.path.exists(\"/root/.primehub/config.json\"):\n",
+    "    ph.config.generate(PRIMEHUB_CLUSTER)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "00a4608c",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# Get config json file information.\n",
+    "!cat /root/.primehub/config.json"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "69a97631",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# Setting the primehub sdk config.\n",
+    "ph = PrimeHub(PrimeHubConfig())"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "db39c554",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# Check the SDK setup is ready.\n",
+    "if ph.is_ready():\n",
+    "    print(\"PrimeHub Python SDK setup successfully\")\n",
+    "    print(\"Current Group:\", ph.primehub_config.current_group)\n",
+    "else:\n",
+    "    print(\"PrimeHub Python SDK couldn't get the group information, please check the configuration.\")"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "14392ecc",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "primehub_job_config = {\n",
+    "    \"instanceType\": \"cpu-1\",\n",
+    "    \"image\": \"tf-2\",\n",
+    "    \"displayName\": \"credit-card-model-training\",\n",
+    "    \"command\": \"\"\"git clone https://github.com/InfuseAI/showcase.git\n",
+    "\n",
+    "pip install kaggle\n",
+    "export KAGGLE_USERNAME={}\n",
+    "export KAGGLE_KEY={}\n",
+    "\n",
+    "cd ~/credit-card-recommendation/data/original\n",
+    "kaggle datasets download -d sajidhussain3/jobathon-may-2021-credit-card-lead-prediction\n",
+    "unzip jobathon-may-2021-credit-card-lead-prediction.zip\n",
+    "\n",
+    "cd ~/credit-card-recommendation/\n",
+    "pip install -r requirements.txt\n",
+    "python main.py\n",
+    "\"\"\".format(KAGGLE_CONFIG['username'], KAGGLE_CONFIG['key'])\n",
```

Comment:
Just confirm, do we need Kaggle config within this notebook? if so, please fill-in pre-req section

Reply:
Done.